config/initializers/git_info.rb

```diff
@@ -0,0 +1,7 @@
+module Git
+    `which git`
+    if $?.success?
```

Comment:
I need to figure out why find_executable crashes on my alpine based container first, but it was my intention in the first place.